server/sonar-server/src/main/java/org/sonar/server/computation/queue/CeQueueInitializer.java

```diff
@@ -47,7 +49,14 @@ public CeQueueInitializer(DbClient dbClient, CEQueueStatus queueStatus, CeQueueC
   }
 
   @Override
-  public void start() {
+  public void onServerStart(Server server) {
+    if (!done) {
```

Comment:
this is code for the sole purpose of testing (in medium tests). I think it's crapy but I couldn't find an alternative